infra/blueprint-test/pkg/utils/transforms.go

```diff
@@ -40,3 +41,12 @@ func GetResultStrSlice(rs []gjson.Result) []string {
 	}
 	return s
 }
+
+// DeleteFromResult deletes given path from result and returns the modified result
+func DeleteFromResult(t testing.TB, pth string, r gjson.Result) gjson.Result {
+	n, err := sjson.Delete(r.String(), pth)
```

Comment:
I'm surprised this can't be done on `r` directly. Parsing in and out of JSON is inefficient.

Reply:
Unfortunately they decided to split that into sjson which doesn't seem to operate on gjson.Result. We could add optimizations for more than one deletes (if there is a use case in the future) to prevent roundtripping.